components/Badge/badge.css

```diff
@@ -0,0 +1,68 @@
+.badge {
+  position: relative;
+}
+.badge-container {
+  display: flex;
+}
+.badge-online,
+.badge-dnd,
+.badge-offline {
+  position: absolute;
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  bottom: 6px;
+  right: 1rem;
+}
+
+.avatar {
+  width: 5.625rem;
+  border-radius: 50%;
+}
+
+.badge-notification i {
+  width: 100px;
+  display: inline-block;
+  text-align: center;
+  vertical-align: middle;
+  position: relative;
+}
+.badge-notification .notification:after {
+  content: "99";
+  position: absolute;
+  background: var(--tertiary-color);
+  height: 1.25rem;
+  top: 1rem;
+  right: 1.5rem;
+  width: 1.35rem;
+  text-align: center;
+  line-height: 1.1rem;
+  font-size: 10px;
+  border-radius: 50%;
+  /* color: white; */
+  border: 1px solid var(--tertiary-color);
+}
+
+.badge-envelope {
+  width: 6.25rem;
```

Comment:
Please try to remain consistent with spacing between your css classes and please remove comments.